style/question.less

```diff
@@ -23,23 +24,28 @@
   .free-response:before,
   .question-feedback,
   .question-feedback:before {
-    font-size: 0.75em;
+    font-size: 0.8em;
   }
   .free-response {
-    padding: 20px 10px;
-    background: #f0f0f0;
+    padding: 20px 40px;
     display: block;
     width: 100%;
     float: left;
-    margin: 1em 0;
-    line-height: 1em;
+    margin: 1.5em 0 .25em 0;
+		color: #555555;
+		font-style: italic;
+    line-height: 1.5em;
+    background: #f5f5f5;
 
     &:before {
       content: "My Answer:";
       display: inline-block;
       float: left;
-      margin-left: -90px;
-      color: #999;
+      margin-left: -145px;
+			font-family: 'Architects Daughter', cursive;
+			font-size: .9em;
```

Comment:
Hmm, @jivewise : should we be using rem's instead of em's (to avoid cascading font size changes)?